music21/variant.py

```diff
@@ -2582,6 +2582,7 @@ def testVariantClassA(self):
         self.assertTrue(v1.hasElementOfClass('Measure'))
 
     def testDeepCopyVariantA(self):
+        # pylint: disable=unnecessary-comprehension
```

Comment:
remove and just use `self.pitchOut(s.pitches)` since it will work even if `s.pitches` eventually returns an iterator (which I don't see as likely...).

Reply:
(and sv.pitches)